projects/kit/components/carousel/carousel.component.ts

```diff
@@ -83,6 +85,9 @@ export class TuiCarouselComponent {
     @Output()
     public readonly indexChange = new EventEmitter<number>();
 
+    @Output()
+    public readonly shifted = new EventEmitter<number>();
```

Comment:
nit: let's call it `shift` as in `click` is the name of the event, not `clicked`

Reply:
fixed